managed/services/supervisord/logs.go

```diff
@@ -59,14 +61,16 @@ type fileContent struct {
 type Logs struct {
 	pmmVersion       string
 	pmmUpdateChecker *PMMUpdateChecker
+	vmParams         *models.VictoriaMetricsParams
 }
 
 // NewLogs creates a new Logs service.
 // n is a number of last lines of log to read.
-func NewLogs(pmmVersion string, pmmUpdateChecker *PMMUpdateChecker) *Logs {
+func NewLogs(pmmVersion string, pmmUpdateChecker *PMMUpdateChecker, vmParams *models.VictoriaMetricsParams) *Logs {
 	return &Logs{
```

Comment:
Can we separate concrete implementation with abstract one? For example vmParams getter? Than if in future we change mechanism that gets us a new vmParams we wont rewrite this place?